assets/js/payment-method-extensions/payment-methods/stripe/payment-request/index.js

```diff
@@ -57,10 +63,10 @@ const PaymentRequestPaymentMethod = {
 	content: <PaymentRequestExpress stripe={ componentStripePromise } />,
 	edit: <ApplePayPreview />,
 	canMakePayment: ( cartData ) =>
-		paymentRequestAvailable(
-			// eslint-disable-next-line camelcase
-			cartData?.cartTotals?.currency_code?.toLowerCase()
-		),
+		paymentRequestAvailable( {
+			currencyCode: cartData?.cartTotals?.currency_code?.toLowerCase(),
+			totalPrice: parseInt( cartData?.cartTotals?.total_price, 10 ),
```

Comment:
Since we get `total_price` using the optional chaining operator we assume that we can get `undefined` as a result. Undefined passed to `parseInt` will get us `NaN`. When we plug the `NaN` to the test used on line 26  `if ( totalPrice < 30 )` we get `False` and continue but we should stop.

Reply:
Good catch. I committed a fallback to `|| 0`.